src/Blocks/BlockButton.h

```diff
@@ -34,7 +34,7 @@ class cBlockButtonHandler :
 		Meta |= 0x08;
 
 		a_ChunkInterface.SetBlockMeta(a_BlockX, a_BlockY, a_BlockZ, Meta, false);
-		a_WorldInterface.WakeUpSimulators(a_BlockX, a_BlockY, a_BlockZ);
+		a_WorldInterface.WakeUpSimulators(Vector3i(a_BlockX, a_BlockY, a_BlockZ));
```

Comment:
I wonder if the compiler would accept the "universal initializer" format here (and in similar places):
```cpp
a_WorldInterface.WakeUpSimulators({a_BlockX, a_BlockY, a_BlockZ});
```